index.html

```diff
@@ -44,8 +44,8 @@ <h3 class="profession">I'am a Software Developer</h3>
               </ul>
           </div>
      </div>
-<section class="card1">
-    <div class="work-title grid1"><h1 id="work-title"> My Recent Works</h1></div>
+<div class="card1">
+    <div class="work-title grid1"><h2 id="work-title"> My Recent Works</h2></div>
     <div class="photo1 grid1"><img src="Img/P_img/photo1.png" alt="photo1"></div>
```

Comment:
- [ ] Instead of using an image here for the horizontal line, you can use an `hr` tag. Also, the line does not extend the full width of the screen as shown in the template. Do well to rectify this.
- [ ] It would be good to add some more space between your first page and your `Recent works` title so as to follow the template.
- [ ] Also, add some space between the horizontal line and the image of your first project as shown in Figma 👍 
- Your design:
![Annotation 2022-03-18 060301](https://user-images.githubusercontent.com/85113567/158941536-7eb473b2-b96d-47c6-b8ea-97f00c6e8de5.jpg)
- Figma design:
![Annotation 2022-03-18 060317](https://user-images.githubusercontent.com/85113567/158941549-6c416ea9-a273-48fe-85e9-1ef45b7c980f.jpg)